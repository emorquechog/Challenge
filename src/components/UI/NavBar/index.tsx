import React from "react";
import Button from "@material-ui/core/Button";
import Menu from "@material-ui/core/Menu";
import MenuItem from "@material-ui/core/MenuItem";
import { Link } from "react-router-dom";

const NavBar = () => {
  const [anchorEl, setAnchorEl] = React.useState(null);

  const handleClick = (event: any) => {
    setAnchorEl(event.currentTarget);
  };

  const handleClose = () => {
    setAnchorEl(null);
  };

  return (
    <div>
      <Button
        aria-controls="simple-menu"
        aria-haspopup="true"
        onClick={handleClick}
      >
        Open Menu
      </Button>
      <Menu
        id="simple-menu"
        anchorEl={anchorEl}
        keepMounted
        open={Boolean(anchorEl)}
        onClose={handleClose}
      >
        <MenuItem onClick={handleClose}>
          <Link to="/register">Register</Link>
        </MenuItem>
        <MenuItem onClick={handleClose}>
          <Link to="/list">List</Link>
        </MenuItem>
      </Menu>
    </div>
  );
};

export default NavBar;
