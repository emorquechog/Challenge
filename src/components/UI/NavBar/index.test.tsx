import React from "react";
import { shallow } from "enzyme";

import NavBar from "./index";

const createWrapper = (props: any) => shallow(<NavBar />);
describe("2 - Block Render Test (snapshot)", () => {

  // CREATE INSTANCE
  const component = createWrapper(null);
  // TESTS
  it("should render correctly", () => {
    expect(component).toMatchSnapshot();
  });
});
