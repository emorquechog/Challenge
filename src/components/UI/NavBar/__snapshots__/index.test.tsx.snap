// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`2 - Block Render Test (snapshot) should render correctly 1`] = `ShallowWrapper {}`;
